rebuild.sh

```diff
@@ -87,18 +194,21 @@ fi
 echo -e "\033[1m$(pwd)\033[0m"
 
 mvnBuildDocker() {
-  local mvnCommand mvnImage crlfDocker mvnVersion
+  local mvnCommand mvnImage crlfDocker mvnVersion mvnDefaultJDKVersion
   mvnCommand="$1"
   crlfDocker="no"
 
   mvnVersion="3.6.3"
+  mvnDefaultJDKVersion=${jdk}
   case ${jdk} in
-    18)
```

Comment:
no, please don't change existing defaults: they are used by existing buildspecs

Reply:
I recently found that Java 18 no longer is available while fixing the Apache Avro build.
https://github.com/apache/avro/commit/12adf0bc57cb1ddcb58a4f6aa41695fb7512140a

So I think that none of the buildspecs should have anything other than an LTS JDK version.

Reply:
> So I think that none of the buildspecs should have anything other than an LTS JDK version

we have no choice: if the release manager did a bad choice that does not last over time, we can't change his choice

Reply:
Yes you are right.

Yet consider: how long after a "bad choice" was made does this project need to be able to support proving the reproducibility of it? I say there should be an plausible end to it.
https://endoflife.date/java

Reply:
experience shows that some projects choose random JDK over releases, which is not an issue for the content (given `-release x` guarantees that there is no issue in the bytecode) but for long term reproducibility, makes environment creation harder.
Notice: harder, not impossible...

Reply:
and we have the statistics of such fragile choices https://github.com/jvm-repo-rebuild/reproducible-central#tool-x-jdk-statistics